test_jsonschema.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2022-2023 Axis Communications AB and others.
```

Comment:
If this is a new file I guess it should read "# Copyright 2023-2024 Ericsson AB."

Reply:
Not sure. A lot of it is copied from another test file, so I'm not sure we could say that Ericsson is the only copyright holder of this file.